lib/register_sources_bods/register/statement_loader.rb

```diff
@@ -56,13 +58,14 @@ def load_statements_children(statements)
       def load_statements_parents(statements)
         new_statements = statements
 
-        level = 1
+        level = 0
         while !new_statements.empty? && (level <= MAX_LEVELS)
           new_statements = load_associated_statements(new_statements.keys, interested_party: true, subject: false).to_h { |r| [r.statementID, r] }.reject { |k, _v| statements.key? k }
 
           next_statement_ids = new_statements.keys
 
           next_statement_ids += statements.values.select { |s| s.respond_to?(:subject) }.map(&:subject).compact.map(&:describedByEntityStatement).compact
+          next_statement_ids += new_statements.values.select { |s| s.respond_to?(:subject) }.map(&:subject).compact.map(&:describedByEntityStatement).compact
```

Comment:
Or something like `[statements, new_statements].map { |e| e..values.select { |s| s.respond_to?(:subject) }.map(&:subject).compact.map(&:describedByEntityStatement).compact }.SOMETHING` (I haven't worked it out).